ads/inabox/inabox-messaging-host.js

```diff
@@ -139,10 +139,13 @@ export class InaboxMessagingHost {
       return true;
     }
 
-    this.registeredIframeSentinels_[request.sentinel] = true;
-    this.positionObserver_.observe(iframe, data => {
-      this.sendPosition_(request, source, origin, data);
-    });
+    const positionObserverHandle = this.positionObserver_.observe(
```

Comment:
Can we rename this to observeUnregisterFn?

Reply:
done